src/program/KeyMapping.h

```diff
@@ -77,6 +79,8 @@ typedef int HotKeyType; enum
     HOTKEY_LOADBRANCH8,
     HOTKEY_LOADBRANCH9,
     HOTKEY_LOADBRANCH_BACKTRACK,
+    HOTKEY_SELECTSTATE_NEXT,
+    HOTKEY_SELECTSTATE_PREVIOUS,
```

Comment:
We would want to add more hotkeys at the end, so that it does not shift existing hotkey mappings

Reply:
Thanks for noticing - I did not realize this. Maybe these enum values could be explicitly numbered to make that more apparant?

Reply:
Ah, just saw there is a comment about this at the top.